report.php

```diff
@@ -14,6 +14,7 @@
 $WPT_PREPARE_DIR = getenv( 'WPT_PREPARE_DIR' );
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' );
 $WPT_REPORT_API_KEY = getenv( 'WPT_REPORT_API_KEY' );
+$WPT_LOGGING = getenv( 'WPT_LOGGING') ? : 'normal';
```

Comment:
Nit-pick, but this could use a space after `'WPT_LOGGING'` for coding standards reasons.

Reply:
oops